pilot/docker/Dockerfile.proxyv2

```diff
@@ -52,5 +52,8 @@ COPY envoy_pilot.yaml.tmpl /etc/istio/proxy/envoy_pilot.yaml.tmpl
 COPY envoy_policy.yaml.tmpl /etc/istio/proxy/envoy_policy.yaml.tmpl
 COPY envoy_telemetry.yaml.tmpl /etc/istio/proxy/envoy_telemetry.yaml.tmpl
 
+COPY stats-filter.wasm /etc/istio/proxy/stats-filter.wasm
+COPY metadata-exchange-filter.wasm /etc/istio/proxy/metadata-exchange-filter.wasm
```

Comment:
so this is the v8 one?

Reply:
This just allows us to use it, it doesn't require it right? The config is from the EnvoyFilter?

Reply:
Yeah these are the v8 based plugins, and yeah there will be option to enable them if they pass stability and perf test.